packages/odyssey-storybook/src/components/odyssey-labs/DataComponents/DataComponents.stories.tsx

```diff
@@ -562,3 +563,41 @@ export const DataStackComponent: StoryObj<DataViewMetaProps> = {
     );
   },
 };
+
+export const ExpandableRowsAndCards: StoryObj<DataViewMetaProps> = {
+  render: function Base(args) {
+    console.log(args);
+
+    const [data, setData] = useState<Person[]>(personData);
+    const { getData, onReorderRows, onChangeRowSelection } = useDataCallbacks(
+      data,
+      setData,
+    );
+
+    const renderAdditionalContent = useCallback(
+      ({ row }: DataTableRenderDetailPanelType) => {
+        return <Box>This is additional content for row {row.id}</Box>;
+      },
+      [],
+    );
+
+    return (
+      <DataView
+        availableLayouts={["table", "grid"]}
```

Comment:
These need to be memoized even if this is a story.

Reply:
Which part? I'm not seeing anything un-memoized that should be, but probably missing it

Reply:
I believe he is referencing the array passed into `availableLayouts` Can probably just do
```js
const availableLayouts = useMemo(() => ["table", "grid"], [])
....
<DataView
  availableLayouts={availableLayouts}
```

Reply:
Even better, just put that array at the top of the component. No reason to memoize a static value :P.